CONTRIBUTING.md

```diff
@@ -31,3 +31,33 @@ To set up the checks, please consult the following instructions:
    necessary tools.
 2. Before making a PR, ensure `npm run lint` and `npm run spellcheck` both
    clear.
+
+## Release Cadence
+
+This chart is intended to give a rough release cadence for iterations for the
```

Comment:
I'm a little confused about what we're doing here... are the releases meaningful? I had expected the overall repo to be under continuous revision, and for "releases" to be deemphasized.

I can see real value in established timelines for certain key "features" (e.g., getting real case studies done, finishing v1 of all diagrams, etc), but that might not be what you're getting at.

To be clear, I don't necessarily object to this, I'm just not sure what a release means for CNO.

Reply:
revised this to try and capture 'key milestones' rather than 'releases'